README.md

```diff
@@ -372,7 +372,8 @@ All lines in between will be waived for rule-X
 // verilog_lint: waive-stop rule-X
 ```
 
-Another option is to use a configuration file with the `--waivers_file` flag.
+Another option is to use a configuration file with the `--waiver_files` flag.
+The flag accepts a single file or a list of files (comma-separated).
```

Comment:
Can you explain how the effects of multiple files are accumulated?
That is, is it equivalent to concatenating files in the order of appearance? or something else?

Reply:
Yes, it is as if they were concatenated. The order doesn't matter for now as waivers can only disable something so there should be no conflicts, but I guess it makes sense to add such a comment in case we add more commands in the future.